WordPress/src/main/java/org/wordpress/android/ui/mysite/ImprovedMySiteFragment.kt

```diff
@@ -296,7 +296,7 @@ class ImprovedMySiteFragment : Fragment(),
                     is ItemUploadedModel.PostUploaded -> {
                         uploadUtilsWrapper.onPostUploadedSnackbarHandler(
                                 activity,
-                                requireActivity().findViewById(R.id.coordinator), true,
+                                requireActivity().findViewById(R.id.coordinator), true, false,
```

Comment:
I was having trouble tracking down why this should be `false` here, but I was also seeing local errors in AS for this file, even though it built, installed, and ran properly :man_shrugging: `Type mismatch: inferred type is () -> [ERROR : <ERROR FUNCTION RETURN TYPE>] but Observer<in Event<SnackbarMessageHolder>!> was expected`.

Not suggesting a change, but curious why this one is `false` (since my code navigation was failing here while reviewing, due to the weird errors I was seeing).

Reply:
> Not suggesting a change, but curious why this one is `false` (since my code navigation was failing here while reviewing, due to the weird errors I was seeing).

Good question @mkevins 👍 
The `ItemUploadedModel.PostUploaded` has only one (that is not test code) [instance creation](https://github.com/wordpress-mobile/WordPress-Android/blob/9a94470259e16d228d73685d0b21e612ec1de665/WordPress/src/main/java/org/wordpress/android/ui/mysite/SiteIconUploadHandler.kt#L61). This is triggered by an `UploadErrorEvent` and always leads to the `isError` if branch of the `UploadUtils.onPostUploadedSnackbarHandler` method.

Reply:
Thanks for the explanation!